src/resources/spi_esp32.cc

```diff
@@ -135,6 +135,7 @@ PRIMITIVE(device) {
     .address_bits     = uint8(address_bits),
     .dummy_bits       = 0,
     .mode             = uint8(mode),
+    .clock_source     = SPI_CLK_SRC_DEFAULT,
```

Comment:
Is that all it takes to upgrade to the newest API?

Reply:
correct.
I changed the title of the PR to reflect it.
done.